### PR TITLE
Rewrite update state script and add comments to client functions

### PR DESCRIPTION
**Pull request by s6thgehr**

### CODE REVIEW

File: packages/yield-router/scripts/updateState.ts [hunk 1]
```diff
@@ -1,57 +1,41 @@
...
-// used for devnet testing
-const defaultStateAddress = "9QxfwoxkgxE94uoHd3ZPFLmfNhewoFe3Xg5gwgtShYnn";
-const stateAddress = new PublicKey(
-  process.env.STATE_ADDRESS ?? defaultStateAddress
-);
+// new update authority
+// very dangerous, isn't it?
+// we can set the authority to some address where we don't have the private key
+// so we can never change the authority ever again
+const newUpdateAuthority = new PublicKey("");
```
Comment (gigileungyingchi) [inline packages/yield-router/scripts/updateState.ts]: I think it's maybe not necessary to include this update in this script here. We are not going to want to always update the authority with a new keypair.

Comment (gigileungyingchi) [reply packages/yield-router/scripts/updateState.ts]: Or maybe we can allow it to be defined externally (e.g. a `process.env.NEW_UPDATE_AUTHORITY`) and skip this update when a new authority is not provided?

File: packages/yield-router/scripts/updateState.ts [hunk 2]
```diff
@@ -1,57 +1,41 @@
...
 );
 
-// used in devnet
-const defaultMint = "tnct1RC5jg94CJLpiTZc2A2d98MP1Civjh7o6ShmTP6";
-const mint = new PublicKey(process.env.MINT ?? defaultMint);
+// mainnet offset bridge wrapped SOL ATA
+const outputYieldAddresses = [
+  new PublicKey("4XTLzYF3kteTbb3a9NYYjeDAYwNoEGSkjoqJYkiLCnmm"),
+];
+const spendProportions = [100];
```
Comment (gigileungyingchi) [inline packages/yield-router/scripts/updateState.ts]: Can we maybe also make it possible to define these parameter externally? So we can update the state using this script without always having to change this script

Comment (s6thgehr) [reply packages/yield-router/scripts/updateState.ts]: That is a very good idea. I will rewrite the script such that the user has to input these parameters in the console.

**Review by gigileungyingchi — APPROVED** [review]

**Pull request merged**